llpc/patch/llpcSystemValues.cpp

```diff
@@ -41,17 +41,10 @@
 using namespace Llpc;
 using namespace llvm;
 
-namespace llvm
-{
-
-namespace cl
-{
-
-extern opt<uint32_t> ShadowDescTablePtrHigh;
-
-} // cl
-
-} // llvm
+// -shadow-desc-table-ptr-high: high part of VA for shadow descriptor table pointer
+static cl::opt<uint32_t> ShadowDescTablePtrHigh("shadow-desc-table-ptr-high",
+                                                cl::desc("High part of VA for shadow descriptor table pointer"),
+                                                cl::init(2));
```

Comment:
Is it possible to describe in the comments why we use 2 as initial value?

Reply:
Why do we use 2?

Reply:
I guess it is required by PAL's dividing internal memory space.

Reply:
done.